test/CMakeLists.txt

```diff
@@ -139,6 +140,9 @@ function(add_umf_test)
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
     set_tests_properties(${TEST_NAME} PROPERTIES LABELS "umf")
+    if(ARG_ENVS)
+        set_tests_properties(${TEST_NAME} PROPERTIES ENVIRONMENT ${ARG_ENVS})
```

Comment:
I'm not a fan of adding such env functionality only for one tests.
Please add `set_tests_properties` just after `add_umf_test` and it should work.
Currently, I try to rewrite from scratch all CMake files to remove such boilerplate.

Reply:
The problem is that the test should be disabled in the previous version of UMF.
In the current version (main branch) the test is removed, but the compatibility test failed without this environment because we need somehow to disable it in the previous versions.

Reply:
So, as I understand it should be enough:
```cmake
if(TARGET ipc_max_opened_limit)
  set_tests_properties(ipc_max_opened_limit PROPERTIES ENVIRONMENT "UMF_MAX_OPENED_IPC_HANDLES=10")
endif()
```
When target is not created then is no need to set it up, it should do the same as your code.

Reply:
Sorry, please disregard my previous comment: I misunderstood your initial feedback.

> So, as I understand it should be enough:
> 
> ```cmake
> if(TARGET ipc_max_opened_limit)
>   set_tests_properties(ipc_max_opened_limit PROPERTIES ENVIRONMENT "UMF_MAX_OPENED_IPC_HANDLES=10")
> endif()
> ```
> When target is not created then is no need to set it up, it should do the same as your code.

Today `add_umf_test` function is generic and allows the setup any environment for any test. Why should we make it test specific like you proposed? What if tomorrow I want to create another test with another environment variable? And actually we will do that in the future for more extensive IPC testing.